proposals/compatibility-mode.md

```diff
@@ -255,6 +255,21 @@ Support for both features is mandatory in core WebGPU.
 
 **Justification**: OpenGL ES 3.1 does not require the relevant f16- or f32-based texture formats (`R16F`, `RG16F`, `RGBA16F`, `R32F`, `RG32F`, and `RGBA32F`) to be color-renderable. While there exist OpenGL ES extensions to enable renderability (`GL_EXT_COLOR_BUFFER_HALF_FLOAT` and `GL_EXT_COLOR_BUFFER_FLOAT`), there are a significant number of devices which lack support for these extensions.
 
+### 23. Disallow multisampled integer textures.
+
+The following integer formats allow multisampling in Core, but not in Compatibility mode:
```

Comment:
Tweak language to clarify this applies to all integer formats, of which the list is just there for reference.